### ISSUE TITLE
Height prop is ignored

### ISSUE THREAD

**Issue by brmscheiner** [AndrewRedican/react-json-editor-ajrm#10]
Hey, great work on this library! This should be a simple fix - I can update the width of the component but not the height.

```
<JSONInput
        id="json-input-unique-class"
        placeholder={this.props.deviceSettings}
        height="580px" // does not work, stays at 635px
        width="480px" // works
        onChange={(e) => { ... }}
/>

**Comment by AndrewRedican** [comment]
Hey @brmscheiner.

Thanks for the kudos.

I really appreciate your feedback. Upon closer examination, I realized component was not rerendering after _any_ property was updated after the intial component mount. *face palm*

This has been fixed on version 2.3.1.

**Comment by brmscheiner** [comment]
Sweet, thank you!
